components/infobox/wikis/apexlegends/infobox_weapon_custom.lua

```diff
@@ -133,9 +133,21 @@ function CustomInjector:addCustomCells(widgets)
 		})
 	end
 
+	if String.isNotEmpty(args.ammotype) then
+		table.insert(widgets, Cell{
+			name = 'Ammo Type',
+			content = {args.ammotypeicon .. ' ' .. args.ammotype}
```

Comment:
can ammotypeicon be nil while ammotype is set?

Reply:
I guess theoretically there could be, but realistically there would be no reason to include the icon and not the text description.
I will amend so that both have to be NotEmpty for the check to pass